Examples/Swift/RBQFRCSwiftExample/MainTableViewController.swift

```diff
@@ -243,58 +244,53 @@ class MainTableViewController: UITableViewController {
 // MARK: -
 
 extension MainTableViewController: FetchedResultsControllerDelegate {
-    func controllerWillChangeContent<T : Object>(controller: FetchedResultsController<T>) {
+    func controllerWillChangeContent<T : Object>(_ controller: FetchedResultsController<T>) {
         self.tableView.beginUpdates()
     }
-    
-    func controllerDidChangeObject<T : Object>(controller: FetchedResultsController<T>, anObject: SafeObject<T>, indexPath: NSIndexPath?, changeType: NSFetchedResultsChangeType, newIndexPath: NSIndexPath?) {
-        
+
+    func controller<T : Object>(_ controller: FetchedResultsController<T>, didChangeObject anObject: SafeObject<T>, atIndexPath indexPath: IndexPath, forChangeType type: NSFetchedResultsChangeType, newIndexPath: IndexPath) {
+
         let tableView = self.tableView
-        
```

Comment:
I've reverted the type optionality for `IndexPath?` and also restored the forced unwrapping of the values inside the method.